god is good/scratch.md

```diff
@@ -19,4 +19,8 @@ niech się weselą na zawsze! \
 Osłaniaj ich, niech cieszą się Tobą, \
 ci, którzy imię Twe miłują.
 <small>13.</small> Bo Ty, Panie, będziesz błogosławił sprawiedliwemu:\
-otoczysz go łaską jak tarczą.
\ No newline at end of file
+otoczysz go łaską jak tarczą.
+
+
+
+na pierwszym Miejscu Bóg
```

Comment:
co to za random entery??

Reply:
zeby nie psuc kubie psalmu czy co tam je

Reply:
no to sie robi
<br><br>

Reply:
co sie robi